benchmarks/dbscan/dbscan.hpp

```diff
@@ -0,0 +1,38 @@
+/****************************************************************************
+ * Copyright (c) 2017-2022 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <ArborX_DBSCAN.hpp>
```

Comment:
That's kinda unfortunate.  The goal would have been to have this in the implementation only.

Reply:
That would be possible if we replace `Implementation` with `std::string`. But I kinda don't want to do that.